lib/Prometheus/Tiny.pm

```diff
@@ -223,6 +231,11 @@ L<Prometheus::Tiny::Shared> for that!
 =head2 new
 
     my $prom = Prometheus::Tiny->new
+    my $prom => Promethus::Tiny->new(default_labels => { my_label => "frob" });
```

Comment:
Tiny doc bug (not `=>`)

Reply:
I stared at this for a really long time thinking "rob they're both `=>` and what do you _want_ them to be" before I saw the one you were actually talking about. Fixed now, thanks!